applications/GeoMechanicsApplication/tests/cpp_tests/custom_elements/test_thermal_integration_coefficients.cpp

```diff
@@ -58,8 +56,9 @@ KRATOS_TEST_CASE_IN_SUITE(ThermalIntegrationCoefficientsCalculator_ReturnsCorrec
     // The expected number is calculated as follows:
     // 2.0 (detJ) * 0.5 (weight) = 1.0
     // cross area is not taken into account
-    KRATOS_EXPECT_NEAR(plane_element.get()->GetProperties()[CROSS_AREA], 0.5, 1e-5);
-    KRATOS_EXPECT_NEAR(calculated_coefficients[0], 1.0, 1e-5);
+    KRATOS_EXPECT_EQ(calculated_coefficients.size(), 1);
+    KRATOS_EXPECT_NEAR(plane_element.get()->GetProperties()[CROSS_AREA], 0.5, Defaults::absolute_tolerance);
+    KRATOS_EXPECT_NEAR(calculated_coefficients[0], 1.0, Defaults::absolute_tolerance);
```

Comment:
I don't think we'd want to assert the value of this property (it is input). If you'd like to be sure that `CROSS_AREA` is not being used for the planar element, we could remove it from the properties before running the calculator:
```cpp
    p_properties->Erase(CROSS_AREA);
```

```suggestion
    KRATOS_EXPECT_EQ(calculated_coefficients.size(), 1);
    KRATOS_EXPECT_NEAR(calculated_coefficients[0], 1.0, Defaults::absolute_tolerance);
```

Reply:
I added this line because before I re-used a function to create the element which uses the default properties. Now it is removed